docs/DeveloperGuide.md

```diff
@@ -307,30 +310,111 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `ArtBuddy`
+and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Delete a customer**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. ArtBuddy shows a list of customers
+2. User requests to delete a specific customer in the list
+3. ArtBuddy deletes the customer and all the customer's commissions
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The customer list is empty.
+
+  Use case ends.
+
+* 2a. The given index is invalid.
+
+    * 2a1. ArtBuddy shows an error message.
+
+      Use case ends.
+
+**Use case: Add a customer**
+
+**MSS**
+
+1. ArtBuddy shows a list of customers
+2. User requests to add a customer and his or her details to the list
+3. ArtBuddy shows the updated list of customers
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. Necessary customer details not given.
+
+    * 2a1. ArtBuddy shows an error message.
+
+      Use case ends.
+
+**Use case: Open a customer**
+
+**MSS**
+
+1. ArtBuddy shows a list of customers
+2. User requests to open a specific customer in the list
+3. ArtBuddy opens the customer and shows the list of commission from this customer
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. The customer list is empty.
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 2a. The given index is invalid.
+
+    * 2a1. ArtBuddy shows an error message.
+
+      Use case ends.
+
+**Use case: Delete a commission**
+
+**MSS**
+
+1. ArtBuddy shows the list of commissions from the opened customer
+2. User requests to delete a specific commission in the list
+3. ArtBuddy deletes the commission and updates the list displayed
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. The commission list is empty.
+
+  Use case ends.
+
+* 2a. The given index is invalid.
+
+    * 2a1. ArtBuddy shows an error message.
+
+      Use case ends.
+
+**Use case: Add a commission**
+
+**MSS**
+
+1. ArtBuddy shows the list of commissions from the opened customer
```

Comment:
```suggestion
1. ArtBuddy shows the list of commissions from the selected customer
```

Reply:
Is it grammatically wrong to use opened haha if that's the case I will change "opened" to "selected" for other instance also

Reply:
I think it's fine we use `opened Customer` since that represents the window with the customer details, but I don't think it really matters since it doesn't affect the meaning and we aren't graded on language